cli/README.md

```diff
@@ -103,6 +103,24 @@ Remove storage by specifying the storage name to be deleted,
 $ kubectl kadalu storage-remove storage-pool-1
 ```
 
+### Remove archived PVCs
+
+Remove all archived pvc's provisioned from the storage-pool/volume.
+
+```
+$ kubectl kadalu remove-archived-pv storage-pool-1
```

Comment:
- enforce `--pool` arg as well to be consistent with `--pvc`

Reply:
Storage pool name is a positional argument here. Should I make it optional? Then what should be the default behaviour? All storage pools?

Reply:
- how about no positional argument with no default behaviour, is that feasible?
- user should provide either `--pool` or `--pool` and `--pvc`